content/en/docs/writing-policies/verify-images/sigstore/_index.md

```diff
@@ -460,6 +460,68 @@ spec:
 
 This enables use cases where, in an enterprise with a private CA, each team has their own leaf certificate used for signing their images, and a global policy is used to verify all images signatures.
 
+
+### Using API Calls to Reference Certificates for Image Verification
+
+Since k8s:// references for certificates are not supported, Kyverno’s context feature can be used to fetch and decode certificates stored in Kubernetes secrets for image verification.
```

Comment:
there is no need to mention that `k8s:// references for certificates are not supported`, its not supported by cosign

We should just mention that certs can be pulled from secrets as follows